tests/deploys.go

```diff
@@ -0,0 +1,83 @@
+package tests
+
+import (
+	"testing"
+	"time"
+
+	"github.com/nspcc-dev/neo-go/pkg/neotest"
+	"github.com/nspcc-dev/neo-go/pkg/util"
+	"github.com/stretchr/testify/require"
+)
+
+type (
+	// RegisterContractInNNSParams is a param struct for RegisterContractInNNS func.
+	RegisterContractInNNSParams struct {
+		// Name is a contract name. Name must not be a top level domain. For instance: netmap.
+		// The final registered domain name will be concatenated with `.neofs`.
+		Name         string
+		ContractHash util.Uint160
+		Email        string
+		Refresh      int64
+		Retry        int64
+		// Expire in milliseconds
+		Expire int64
+		TTL    int64
+	}
+)
+
+// NewRegisterContractInNNSParams is a constructor for RegisterContractInNNSParams.
+func NewRegisterContractInNNSParams(name string, contractHash util.Uint160) RegisterContractInNNSParams {
+	return RegisterContractInNNSParams{
+		Name:         name,
+		ContractHash: contractHash,
+		Email:        "ops@nspcc.ru",
+		Refresh:      3600,
+		Retry:        600,
+		Expire:       int64(365 * 24 * time.Hour / time.Millisecond), // 1 year.
+		TTL:          3600,
+	}
+}
+
+func nnsContractInvoker(t *testing.T, e *neotest.Executor) *neotest.ContractInvoker {
+	nnsHash, err := e.Chain.GetContractScriptHash(1)
```

Comment:
i think @roman-khimov prefers https://github.com/nspcc-dev/neofs-contract/blob/db6dbb9fa0116f95b7532c84266c4ca84cb94227/rpc/nns/hashes.go#L34 or at least const from this package

Reply:
So do I, but I have nothing to suggest as a `ContractStateGetter` for InferHash func.

Reply:
nns.ID is still relevant here.

Reply:
> nothing to suggest as a ContractStateGetter for InferHash func

wasnt `e.Chain` good enough?

Reply:
Unfortunately no, neotest Chain is not good enough